wbstack/src/Settings/LocalSettings.php

```diff
@@ -515,19 +511,39 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
 $wgWBClientSettings['thisWikiIsTheRepo'] = true;
 $wgWBClientSettings['repoUrl'] = $GLOBALS['wgServer'];
 $wgWBClientSettings['repoSiteName'] = $GLOBALS['wgSitename'];
-$wgWBClientSettings['repositories'] = [
-    '' => [
-        // Use false (meaning the local wiki's database) if this wiki is the repo,
-        // otherwise default to null (meaning we can't access the repo's DB directly).
-        'repoDatabase' => false,
-        'baseUri' => $wgWBRepoSettings['conceptBaseUri'],
-        'entityNamespaces' => [
-            'item' => 120,
-            'property' => 122,
-        ],
-        'prefixMapping' => [ '' => '' ],
-    ]
-];
+
+$wgWBRepoSettings['entitySources'] = array (
```

Comment:
[* (repo+client) (T257262) The entitySources setting is no longer optional, but required.](https://github.com/wikimedia/Wikibase/blob/master/RELEASE-NOTES-1.37)
[* (client) (T285471) repositories has been removed.](https://github.com/wikimedia/Wikibase/blob/master/RELEASE-NOTES-1.37)